mysql-test/suite/tokudb.alter_table/t/mod_enum.test

```diff
@@ -50,7 +50,8 @@ ALTER TABLE test_enum MODIFY COLUMN col2 ENUM('value1','value2','value4');
 
 set tokudb_disable_hot_alter=1;
 set tokudb_disable_slow_alter=0;
+--error 1265
```

Comment:
Perhaps, most of the TokuDB tests use raw error #s and honestly, because MySQL does not print the ER_ in the error message, in almost every case where I need to deal with an error, I need to go look it up regardless to either cross reference the ER_ to # or # to ER_.

I would rather keep the # in tests that already had a # rather than create further semantic differences between 5.6 and 5.7, then maybe one day push changes through from 5.6 -> 5.7 that change to whichever standard we want.

That being said, I know there are tests in this PR that did use ER_ that I changed to # without really thinking about it.